en/docs/Administer/ProductAdministration/ManagingUsersAndRoles/managing-permissions.md

```diff
@@ -1,175 +1,158 @@
 
--   The **Admin** user and **Admin** role is defined and linked to each other in the `user-mgt.xml` file, stored in the `<PRODUCT_HOME>/repository/conf/` directory as shown below.
+By default, every WSO2 product comes with the following Users, Roles and Permissions configured:
 
-    ``` java
+**Users:**
+
+- **Admin** - Has all the permissions in the system enabled by default. Therefore, this is a super tenant, with all permissions enabled. By default, the admin user is assigned to both the **admin** and the **everyone** roles.
+
+!!! info
+
+    -   The **admin** user and **admin** role is defined and linked to each other in the `user-mgt.xml` file, stored in the `<PRODUCT_HOME>/repository/conf/` directory as shown below.
+
+        ``` java
         <AddAdmin>true</AddAdmin>
         <AdminRole>admin</AdminRole>
         <AdminUser>
-             <UserName>admin</UserName>
-             <Password>admin</Password>
+            <UserName>admin</UserName>
+            <Password>admin</Password>
         </AdminUser>
-    ```
+        ```
+        These properties can be customized by configuring the `<PRODUCT_HOME>/repository/conf/deployment.toml` file. For more information, see [Configuring the System Administrator]({{base_path}}/Administer/ProductAdministration/ManagingUsersAndRoles/ManagingUserStores/configuring-the-system-administrator) 
 
--   The **Admin** role has all the permissions in the system enabled by default. Therefore, this is a super tenant, with all permissions enabled.
+    -   The **admin** role has all the permissions in the system enabled by default. Therefore, this is a super tenant, with all permissions enabled.
 
-You will be able to log in to the management console of the product with the **Admin** user defined in the `user-mgt.xml` file. You can then create new users and roles and configure permissions for the roles using the management console. However, note that you cannot modify the permissions of the **Admin** role. The possibility of managing users, roles and permissions is granted by the **User Management** permission. F or more information, s ee [Configuring the User Realm](https://docs.wso2.com/display/Carbon443/Configuring+the+User+Realm) .
+**Roles:**
 
-### Description of role-based permissions
+By default, all WSO2 products have the following roles configured: 
 
-!!! note
-Note that the descriptions given in this document only explains how permissions control access to operations available on the management console.
+-   **admin** - Provides full access to all features and controls. By default, the admin user is assigned to both the admin and the `Internal/everyone` roles.
+-   **Internal/everyone** - This is a conceptual role that is used to group all the users (across the user stores) together. When you create a new user, automatically the user belongs to the `Internal/everyone` role. It does not include any permissions.
```

Comment:
Shall we use the word "pre defined" in place of "conceptual" ?

Reply:
Shall we also include the sentence, "This role can be used to identify all logged in users"